packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -73,7 +79,10 @@ export function PanelChrome({
   padding = 'md',
   title = '',
   description = '',
-  displayMode = 'default',
+  collapsible = false,
+  isOpen,
+  displayMode = collapsible ? 'transparent' : 'default',
```

Comment:
why does the collapsible impact displayMode ?

Reply:
this is to hide PanelChrome's top border line, because it's wrapped in Collapsed